src/algebra/star/basic.lean

```diff
@@ -3,7 +3,6 @@ Copyright (c) 2020 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
-import algebra.big_operators.basic
```

Comment:
Do you want to add a `assert_not_exists finset` to this file if your goal is to ensure it predates finsets?

Reply:
There's an annoying feature that you can't have two files with `assert_not_exists` for the same declaration, if there is any downstream file importing both.  So I'd like to wait until #17768 and #17791 are also merged, and then put the  `assert_not_exists finset` into `data.real.basic`.

Reply:
> There's an annoying feature that you can't have two files with `assert_not_exists` for the same declaration, if there is any downstream file importing both

Wait, why not? I thought I fixed this/

Reply:
Oh, I didn't realise it had been fixed!